src/ngraph/builder/split.cpp

```diff
@@ -74,3 +99,24 @@ NodeVector builder::split(const Output<Node>& value, size_t split_parts, int axi
     std::vector<size_t> length_parts(split_parts, length_axis_to_split / split_parts);
     return split(value, length_parts, axis_to_split);
 }
+
+NodeVector builder::opset1::split(const Output<Node>& value,
+                                  const std::vector<size_t>& length_parts,
+                                  int64_t axis)
+{
+    const auto axis_node = op::Constant::create(element::u64, Shape{}, {axis});
+    const auto length_parts_node =
+        op::Constant::create(element::u64, Shape{length_parts.size()}, length_parts);
+    const auto variadic_split =
+        std::make_shared<ngraph::opset1::VariadicSplit>(value, axis_node, length_parts_node);
```

Comment:
`ngraph::` namespace prefix is unnecessary. We're already `using namespace ngraph;`

Reply:
Because of namespace `opset1` inside namespace `builder` in order to define `VariadicSplit:v1` full namespace path is required